data/mods/FictonalWeapons/fic_weapons.json

```diff
@@ -35,7 +35,7 @@
     "skill": "pistol",
     "ammo": "454",
     "weight": 1500,
-    "volume": 2,
```

Comment:
![image](https://user-images.githubusercontent.com/30208401/71314032-886a5200-240f-11ea-9f69-88e07dc9eb02.png)
it looks like this doesn't quite work

Reply:
That's due to the built in mod.
![image](https://user-images.githubusercontent.com/42699974/71314158-61545600-23f8-11ea-8595-9014ac90e4da.png)